lti_consumer/templates/html/lti_iframe.html

```diff
@@ -7,4 +7,5 @@
     allowfullscreen="true"
     webkitallowfullscreen="true"
     mozallowfullscreen="true"
+    allow="microphone *; camera *; midi *; geolocation *; encrypted-media *"
```

Comment:
Would you also like to lower the permissions here? 
`microphone *; camera *`

Reply:
No, I updated my comment on the ticket, after a brief discussion with someone on the security team, we are going to begin with allowing the same access as they had in chrome v63. I've asked to have a larger discussion with the security group and we can revisit this after their recommendations.